lib/ansible/cli/arguments/option_helpers.py

```diff
@@ -118,6 +118,24 @@ def inner(value):
     return inner
 
 
+def forbidden_characters(nope):
+    ''' ensure option does not have any characters in a list'''
+
+    if not isinstance(nope, list):
+        raise Exception(f"Expected list but got {type(nope)}")
+
+    def inner(value):
+        for x in nope:
+            if x in value:
+                raise Exception(f"Invalid character '{x}' found in '{value}'")
```

Comment:
perhaps, use a better error? like `ValueError`?

Reply:
i was using AnsibleOptionsError  but that didn't seem to work for argparse, will do ValueError